### ISSUE TITLE
Accessibility Fixes for Visually Impaired and Blind Users

### ISSUE THREAD

**Issue by lemoness** [HabitRPG/habitica#6397]
From an email:

"I am blind and have recently stubled across this site and game and think it is both interesting and a real help to my own goals as well as fun. However, the site has a number of problems for the visually impaired and blind like myself.

For example, a number of buttons on the pages are not labeled and are simply assigned a number, for example, When my program (JAWS) one of the major computer screenreaders sets on the button immediately after the text edit windows for new habits, dailies or to-dos, it simpley says “unlabled four button”

I also cannot alter things in the more elaborate ways such as assigning dailies to only certain days of the week or setting up habits so they are only one-sided.

I also have no means to tell what potions are highlighted when hatching eggs so I can’t hach them without sighted help.

Most of these issues are easily fixable, and I’ve worked with others on sites, (on the observation and critique angle, not coding) to fix them for games just as complex as this one, and I would really like it if some of these problems could be addressed.

Also, if the graphics for the avatars and backgrounds could be named either on radio buttons, combo boxes, or other means of selecting them so that people like me could still have a basic idea of what they are rather than simply hearing a number or unlabled graphic, that would be great. Although I don’t see, I used to and I would know what blue wizard robe, would look like if it came up or short red hair."

**Comment by ghost** [comment]
I will look into this for you.

**Comment by citrusella** [comment]
Random comment: I believe fixing labeling issues might also solve problems with some speech recognition programs that I raised on the accessibility trello card a bit back, since it looks (to my untrained eye) like the button names my computer's speech program uses are the same labels a screen reader would use to identify content.

**Comment by ghost** [comment]
citrusella: that was my thought as well. I am having issues getting my dev environment setup (thanks holidays). Once the chaos is over I will dive deep and try to find a good solution for this issue.

**Comment by daggerdragon** [comment]
I have some free time this weekend and I will see what I can do to help with labeling, ARIA compliance, etc as well.

**Comment by lemoness** [comment]
This is a duplicate issue, so I am closing it!

**Comment by citrusella** [comment]
@lemoness May I ask which issue this is a duplicate of? I want to follow it, but I'm having trouble finding it.

**Comment by lemoness** [comment]
Oh dear, I'd have to do a bit of hunting -- it was one of the twenty-seven that I reviewed yesterday. If you search for "JAWS" it might come up!

**Comment by citrusella** [comment]
Aha! It seems to be #6426!
